cmd/collectors/simple/simple.go

```diff
@@ -1,3 +1,5 @@
+// Copyright NetApp Inc, 2021 All rights reserved
```

Comment:
do we need to keep on updating year as well in existing files?

Reply:
good question - i'd prefer not